tests/ui/parser/issues/issue-68987-unmatch-issue-2.stderr

```diff
@@ -6,14 +6,5 @@ LL | async fn obstest() -> Result<> {
 LL |     let obs_connect = || -> Result<(), MyError) {
    |                                               ^ mismatched closing delimiter
 
-error: unexpected closing delimiter: `}`
-  --> $DIR/issue-68987-unmatch-issue-2.rs:14:1
-   |
-LL |     let obs_connect = || -> Result<(), MyError) {
-   |                                               - missing open `(` for this delimiter
```

Comment:
this is needed, pointing the position for `)` may help user figure out `)` should be `>`.